### ISSUE TITLE
Brainstorming Session

### ISSUE THREAD

**Issue by thatguychrisw** [Slimfit/climatepatch#1]
Figured we could start a thread to get some ideas flowing.

**Comment by Slimfit** [comment]
I added a couple tags. Id like to consolidate the ideas that we receive into a backlog that we can start prioritizing. If we gain traction with 1 small team and 50 ideas, it will be helpful to see what ideas are in highest demand vs the impact/value they would provide vs the LoE. Ill add your ideas to the [Trello](https://trello.com/b/AHP33SW7/sprint-0) board tomorrow.

**Comment by artwel** [comment]
IDEA: Monitor home power consumption and show to user.
Send amount of electricity consumption recorded from power sockets and show statistics (living room consumption, bedroom consumption, socket 1, socket2, ...). User can see which device waste a lot of energy in one view.

**Comment by thatguychrisw** [comment]
@Slimfit Sounds good, some of these ideas could be started and worked on gradually like the curated set of climate change studies for instance.

@artwel That's a great idea.  To add to that idea we could use a microcontroller like the Raspberry Pi to actually provide the data and do it in realtime.

**Comment by indigocarmen** [comment]
@artwel That is a cool idea!

What about a botnet that reduces monitor brightness level by like 10%? Across a massive network we could actually see a fairly large energy difference lol. 

I also like the idea of designing a site that is a repo of studies/white papers/etc. that u/ch0chi mentioned. A way to educate head in the sand type to how important this is to our species future.

**Comment by cqcwillard** [comment]
@indigocarmen the more I thought about the curated list of studies, the more I think that's almost a "must-do" at this point.  

We already see our president beginning to place climate change deniers in the EPA; it would be a great idea to provide easy access to data for quicker refutations of claims.  I think a GitBook would be appropriate so it could be easily curated, and appended to.

**Comment by cqcwillard** [comment]
Here's some resources I found earlier this morning.

https://github.com/caesar0301/awesome-public-datasets#climate-weather

**Comment by NicholasGWK** [comment]
Should we be adding new ideas here or in a separate PR? Seems like a good idea!

My idea: Provide easily accessible consumer data for alternative energy. Comparison of local solar installers, wind, grants / government subsidies, community solar programs, solar bonds/investments etc. IMO the only true path to solve climate change is reducing our need for fossil fuel energy. The easier/faster we can make it for people to transition to renewables the betters!

**Comment by cqcwillard** [comment]
@NicholasGWK you can submit a PR as well, others have done that.

My team at work uses GitHub issues to manage ideas and backlog items and it works really well.  Ideas typically end up including discussions. Once we get a few ideas that we want to really run with then, I think we would get more value out of the PRs.

For now, I think the "issues" section provides easier means to have an open dialogue.  Your thoughts?

**Comment by Slimfit** [comment]
@NicholasGWK @cqcwillard  Im good with this approach. I added it to the [docs ](https://github.com/Slimfit/climatepatch/blob/master/Phase1.md)for new comers.

**Comment by raindogg** [comment]
A big concern will be the reduction of <a href="http://www.npr.org/2016/11/10/501597652/fact-check-donald-trumps-first-100-days-action-plan">federal regulations</a> currently in place. A resource that listed what those regulations are and what the impact would be if they were cut might be useful. Maybe work it into the simulator idea somehow?

**Comment by CSCG** [comment]
I came from the reddit thread -- and I strongly propose a nice simple UI aggregation site of studies, data, and datasets showing implications and proof of climate change. 

I also think it may be useful to have an open Re:dash instance with all datasets in there; or related ones, to better allow a consumer-level person who is interested to delve deeper into the data. 

Further - I own the domain 'positiveparty.us' if any political implications or reach can be had with this project. 

'We're positive about the future - we're positive we need change.'

**Comment by DaveMoran** [comment]
One thing we may want to think about as well is adding some sort of gamification to the service.

Jumping off of what @artwel said, maybe we can give badges to users who reduce their energy consumption over a certain amount of time.  This would give users a way of sharing their accomplishments.

I've used the API from cred.ly before (https://credly.com/), however I'm sure that there are other services out there we could use.

**Comment by gigerbytes** [comment]
the gamification thing might work (its already being done by joulebug.com - but they focus on stuff that will save you money)

**Comment by indigocarmen** [comment]
Looking around it does look like there are already a ton of different sites out there that host a ton of information on this topic. While gathering papers and information is great, what would we be doing differently? What would make us stand out versus NASA? 

I think the gamification is a cool idea. What about partnering up with a company to provide discounts based on the amount of power lessened?

**Comment by knightjdr** [comment]
Displaying socket or device power usage and tracking over time is a great idea. I know many people that would appreciate this for itself. I'd be happy to help with this.

Coupling it with discounts could be exploited though: people registering and intentionally using a lot of power one week so that they can decrease it the next for some sort of advantage for example. It could be incorporated but needs some thought on how to prevent exploitation.

**Comment by raindogg** [comment]
It might be beneficial to reach out to some existing advocacy groups, like Earth Justice or Sierra Club.

**Comment by kvnkho** [comment]
@thatguychrisw I like the idea of providing alternative products that do less environmental harm. I picture it as a website like "Eat this, not that". They show two similar dishes with drastically different nutrition values, but not too different taste wise (I think). In the same regard, it would be good as a consumer if I could know how to easily reduce my footprint, just by changing my behavior a bit. Rather than convincing people to care, maybe it's better to empower the ones that do and hope that behavior proliferates.

**Comment by indigocarmen** [comment]
@kvnkho I love that idea! What about teaming up with TWIT to do a weekly podcast about energy efficient devices?

**Comment by indigocarmen** [comment]
We could also have a debunking myths section too?

**Comment by kvnkho** [comment]
@indigocarmen I'm not familiar with TWIT but the podcast idea does sound great along with your debunking myths section. I have never really explored environmental podcasts so I don't know if something similar is out there. Any input from people who do would be cool. 

I was imagining a web portal, where for example, I could compare how many times I need to use an eco-bag to make it more worth it than plastic bags. Seems a podcast and site could really go hand in hand.

**Comment by cqcwillard** [comment]
@kvnkho I like the idea of a podcast a lot.  I think the application ideas are great, but I think what will prove to add the most value in terms of raising awareness is simply providing quick and easy access to factual information regarding climate change.  

ClimateHoaxBusters anyone? :)

**Comment by DaveMoran** [comment]
Kind of boiling everything down here, I'm seeing that we all kind of agree on:

1. Creating a website with accompanying content (Blog roll, podcasts, etc.).
2. Creating a tool that will allow users to create a baseline for their carbon footprint and periodically give them tips for reducing it.
3. Create an achievement system so users can share with their friends the progress they've made.

We should also think about the local markets point of view.  Should we create a map that displays your nearest farmers market?  Should we allow users to upload receipts to give them tips on which foods to avoid and which to substitute?

**Comment by DaveMoran** [comment]
Also side note:  It looks like Nest's API actually has an 'Energy Read' portion of their API.  If anyone knows someone that has Nest it could be something worth looking into

**Comment by indigocarmen** [comment]
Oh I like that! Each week/month we take a Climate Hoax and put it through the ringer? 

@kvnkhoTWIT is a podcast channel dedicated to technology, but with other branches such as Law. We could do This Week in Climate Hoaxes (TWICH)?

@DaveMoran The farmer's market idea would be good, but how would we do the reciept? Stores have such variable naming conventions? The carbon footprint idea could be similar to the "How Many Slaves do You Use" tool. http://slaveryfootprint.org/

**Comment by Slimfit** [comment]
I like the podcast idea. This is another project that can be run in parallel to others. I moved this repository to an organization so we can spin up additional repos that are owned by other team leads. Example climatepatch/podcasts. If anyone wants to run with this idea, start the discussion in a new issue and add progress to the [podcast page](https://github.com/climatepatch/formation/blob/master/Solutions/Podcast.md).

**Comment by DaveMoran** [comment]
@indigocarmen Yeah that's the hard part of that.  Only other thing i can think of is barcode scanning.  Maybe a  'Check if this food is green' tool so that if someone scans a product that isnt green, we can provide alternatives

**Comment by Slimfit** [comment]
Heres another idea from [reddit](https://www.reddit.com/r/climatepatch/comments/5cf4in/some_brainstorming/):

> Some people have already been suggesting to raise awareness to the issue. Though I agree that it's not a bad idea, I think there's already a lot of awareness in the world about climate change.
>
>The biggest problem people face, is that they have the feeling that they can't change anything about it.
>
>What I would suggest, is to create a website where people can participate to small challenges, like going vegetarian for 1 week, or go to work by bike, or other small stuff like that. It's things that people can do, that will have great results when a lot of people commit to it.
Examples of challenges would be:
* go vegetarian for a week
* go by bike to work/school
* buy only local stuff
* try to reduce your waste for a week
>
>We could make it something social, by making people share ideas and pictures of them doing these challenges. This would further motivate new people to join, since they see all kinds of people help.
You could start new challenges, and send them to people by mail, to make them come back. Go for extra hard challenges, etc.
>
>I would make a social platform where people can work together on challenges, so they get the feeling that they're making a change to the world.

With this idea and all the others in this thread, this app almost sounds like an accounting app. A users balance starts at 0. Poor choices that have a negative impact are expenses. Good choices with a positive impact are income. 

A minimum viable product might look something like:
* User can create an account
* User can manually add "expenses". Example: provide a way for them to input an estimated amount of miles driven per week. With this estimation plus their vehicles make and model, we can debit their "account" every week automatically.
* User manually participates in challenges to keep their account in the green. See the quote above for examples.
* The challenge is to stay carbon neutral.

Enhancements that can build on this MVP:
* Receipt scanning can automate incomes and expenses. If you go buy a bunch of local produce, you'll have banked an income. If you purchased a bunch off beef and some motor oil, it will log an expense. 
* Integration with thermostat APIs to expense based on energy usage. 
* etc.

This feels like a game to the user, but if they play enough it will give them the information to see what is the smallest change they have to make to be carbon neutral. Perhaps you can still eat hamburgers occasionally but only if you sell your diesel truck and buy an electric car.

**Comment by Slimfit** [comment]
I liked this idea so I ran with it to see what it would look like. The [demo ](http://carbonbooks.net/) has a simple "feed" of transactions and the positive or negative impact each have on that users individual carbon footprint. The real thing could log positive transactions as challenges completed by the user. The negative transactions could also be logged manually, but there are lots of ideas weve been tossing around that could be built into stand alone services that this api consumes (ex: Nest API or receipt scanning). 

Ignore the registration stuff. Its not really part of the demo but I didn't want to waste time removing it.

**Comment by indigocarmen** [comment]
Oh that looks good! I wonder if we could incentive this? Such as if they are positive 30 they could be entered into a drawing for a gift card to somewhere? We could probably work with some of our eventual sponsors to get gift cards. 

Also, how would we determine scores? You have some great examples but how do we accurately determine what each action is? After all, selling your Ford F250 and buying a Prius probably has more weight than eating a veggie meal lol

**Comment by Slimfit** [comment]
Yeah scores are the hard part. I think what you guys are working on in #12 would help. If we were able to assign numbers/scores to businesses somehow, we could tap into that when a user interacts with the business. I have some ideas Ill flesh out over the week in the demo and see what ya'll think.

**Comment by jgoslow** [comment]
Love the scoring idea - can I suggest it as a sprint-style learning tool instead of the core of the platform? It has a lot of immediate appeal, but high interaction quotient and very little in the way of long-term incentive. Like "Use for a week and get your score! Use for a month to improve your habits!"

To extend the business idea, what if there were scores for companies, and you could look up any company and create your own list of approved-vendors? 

Our biggest power is purchasing - so there's a lot of power and utility here.

**Comment by indigocarmen** [comment]
> To extend the business idea, what if there were scores for companies, and you could look up any company and create your own list of approved-vendors?

Lol, you're thinking right in line with the rest of us there brother! I think I saw you in the other Issue regarding politician and Company scoring and trying to keep these entities honest regarding climate change .

**Comment by jgoslow** [comment]
Just had a little epiphany.  Extending on the idea of gamification and patches.  What if we utilize the scout-patch theme and give out patches for certain actions.  So you collect "climate patches". Make sense?
